token-lending/program/tests/borrow.rs

```diff
@@ -10,13 +10,21 @@ use spl_token_lending::{
 };
 
 const LAMPORTS_TO_SOL: u64 = 1_000_000_000;
+const FRACTIONAL_TO_USDC: u64 = 1_000_000;
 
 #[tokio::test]
 async fn test_borrow_quote_currency() {
-    // Supply values are chosen to fill two orders in the dex market
-    const INITIAL_SOL_RESERVE_SUPPLY_LAMPORTS: u64 = 850 * LAMPORTS_TO_SOL;
-    const INITIAL_USDC_RESERVE_SUPPLY_FRACTIONAL: u64 = 1_878_925_000;
-    const USER_SOL_COLLATERAL_LAMPORTS: u64 = INITIAL_SOL_RESERVE_SUPPLY_LAMPORTS / 2;
+    // Using SOL/USDC max 3 bids:
+    //  $2.199,  300.0 SOL
+    //  $2.192,  213.3 SOL
+    //  $2.190, 1523.4 SOL
+    //
+    // Collateral amount = 600 SOL
+    // Borrow amount = 2.199 * 300 + 2.192 * 213.3 + 2.19 * 86.7 = 1,317.1266 USDC
```

Comment:
This is a great test -- if only we had easier ways of managing the dex orderbooks.